server/src/mcr/admin/admin-controller.spec.js

```diff
@@ -35,7 +41,7 @@ describe('Admin controller testing', function () {
 
     req.headers['authorization'] = 'Bearer ' + adminToken
     res = controller.headerCheck(req)
-    should.not.exist(res)
+    // should.not.exist(res)
```

Comment:
are we testing something here?

Reply:
@bryan-gilbert This is related to the points I would like to further discuss about, therefore my first commit fixes the other issues but it doesn't address this one directly yet. Thank you!